mail_activity_board/models/mail_activity.py

```diff
@@ -52,7 +52,9 @@ def open_origin(self):
 
     @api.model
     def action_activities_board(self):
-        action = self.env.ref("mail_activity_board.open_boards_activities").read()[0]
+        action = (
+            self.env.ref("mail_activity_board.open_boards_activities").sudo().read()[0]
+        )
```

Comment:
```suggestion
        action = self.env["ir.actions.actions"]._for_xml_id(
            "mail_activity_board.open_boards_activities"
        )
```

FTR, this also does the trick and it's slightly better security wise.